pkg/api/routes.go

```diff
@@ -134,6 +162,104 @@ func (rh *RouteHandler) SetupRoutes() {
 	}
 }
 
+func (rh *RouteHandler) UserInfoLogic(ctlr *Controller) rp.CodeExchangeUserinfoCallback {
```

Comment:
Should be after comment `// Method handlers`, and should be documented, just like the other RouteHandlers below. But let's discuss these endpoints with @rchincha and @peusebiu first.